pandas/core/frame.py

```diff
@@ -4764,36 +4764,49 @@ def _gotitem(self, key, ndim, subset=None):
         return self[key]
 
     _agg_doc = dedent("""
+    Notes
+    -----
+    The default behavior of aggregating over the axis 0 is different from
+    `numpy` functions `mean`/`median`/`prod`/`sum`/`std`/`var`, where the
+    default is to compute the aggregation of the flattened array (e.g.,
+    `numpy.mean(arr_2d)` as opposed to `numpy.mean(arr_2d, axis=0)`).
+
+    `agg` is an alias for `aggregate`. Use the alias.
```

Comment:
I am not sure this is relevant, we *always* specify an axis, so by-definition you always have column-by-column or row-by-row aggregations, ``axis=None`` is simply not possible.

Reply:
maybe adding some expl would help here.

Reply:
Is it better now?